programs/ssl/ssl_base64_dump.c

```diff
@@ -206,5 +234,7 @@ int main( int argc, char *argv[] )
         }
     }
 
+    printf( "Finish. Found %zu base64 codes\n", b64_counter );
```

Comment:
I'm not sure we can use `%zu` on all our supported platforms. Might need to use inttypes.h printing macros, or something uglier.

Reply:
I changed all printed size_t values into uint32_t